services/export/tests/unit/shared/transactionsExport.test.js

```diff
@@ -507,3 +405,32 @@ describe('Test resolveChainIDs method', () => {
 		expect(Object.getOwnPropertyNames(response).length).toBe(0);
 	});
 });
+
+describe('Test normalizeBlocks method', () => {
+	it('should return a blocks normalized when called with valid blocks', async () => {
+		const normalizedBlocks = await normalizeBlocks(blocks);
+		const expectedResponse = [
+			{
+				blockHeight: 15,
+				blockReward: 0,
+				date: '2022-11-17',
+				time: '11:52:28',
```

Comment:
Should we get all this values from blocks rather than hardcoding? 

```suggestion
				blockHeight: blocks[0].height,
				blockReward: block[0].reward,
				date: dateFromTimestamp(block[0].timestamp),
				time: timeFromTimestamp(block[0].timestamp),
```